pyi.py

```diff
@@ -246,6 +246,31 @@ def _check_platform_check(self, node: ast.Compare) -> None:
         else:
             self.error(node, Y007)
 
+    def visit_FunctionDef(self, node: ast.FunctionDef) -> None:
+        self.generic_visit(node)
+
+        for i, statement in enumerate(node.body):
+            if i == 0:
+                # normally, should just be "..."
+                if isinstance(statement, ast.Pass):
+                    # should disable this if https://github.com/python/typeshed/pull/1009 isn't
+                    # accepted
```

Comment:
It's not obvious but it's doable. Look at flake8-bugbear's B9 checks. Those are disabled by default.